ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -294,3 +295,41 @@ def ocs_install_verification(timeout=600, skip_osd_distribution_check=False):
                 item for item in crush_rule['steps'] if item.get('type') == 'zone'
             ], f"{crush_rule['rule_name']} is not with type as zone"
         log.info("Verified - pool crush rule is with type: zone")
+
+
+def add_capacity(capacity):
+    """
+   Add storage capacity to the cluster
+   Args:
+        capacity(int): Size of the storage to add as number of deviceSets
+   Returns:
+        boolean : Returns True if all OSDs are in Running state
+   """
+    ocp = OCP(namespace=defaults.ROOK_CLUSTER_NAMESPACE, kind=constants.STORAGECLUSTER)
+    sc = ocp.get()
+    device_set_count = sc.get('items')[0].get('spec').get('storageDeviceSets')[0].get('count')
+    capacity_to_add = device_set_count + capacity
+
+    # adding the storage capacity to the cluster
+    params = f"""[{{"op": "replace", "path": "/spec/storageDeviceSets/0/count", "value":{capacity_to_add}}}]"""
+    ocp.patch(
+        resource_name=sc['items'][0]['metadata']['name'],
+        params=params,
+        format_type='json'
+    )
+    pod = Pod()
+    pod.ocp.wait_for_resource(timeout=180, condition=constants.STATUS_RUNNING,
```

Comment:
may be a good idea to increase the timeout to 300 to be on safer side.

Reply:
will do